### ISSUE TITLE
"Page not found" on fave.

### ISSUE THREAD

**Issue by IngwiePhoenix** [FurryDigital/FurryDigital#31]
Hey @SlvrEagle23 
Wanted to fave your sona, only to run into what seems to me like a wrong route :)

URL: https://furry.digital/view/fav/1/key/DqEINPof1O

Result is "Page not found".

**Comment by SlvrEagle23** [comment]
Faves and unfaves are fixed by a few template updates and routing changes. Turns out on my local machines running their VMs, the case-sensitivity of the filesystem is "inherited" from the parent OS, in this case OSX, which isn't case sensitive by default. This causes problems when uploading to an Ubuntu server which IS of course case sensitive, when the route configuration isn't set up exactly right.

Long story short...it's fixed. :smile: This also fixed comments' routing issue but not other issues with the comment system, which I am presently working on.

**Comment by IngwiePhoenix** [comment]
Lolsome, same here. I tend to run into the same, just that I don’t use a VM locally  - but the same base OS, OS X.

Glad its fixed! Gonna go back and throw that belated fave :)
